internal/services/servicebus/servicebus_queue_data_source_test.go

```diff
@@ -40,10 +42,18 @@ func TestAccDataSourceServiceBusQueue_basic(t *testing.T) {
 	})
 }
 
+// TODO remove test in 5.0
```

Comment:
These todo comments can be removed as the `features.FivePointOh` flag will be what we use to find things we have to remove. It's technically fine but the comment is repeating `features.FivePointOh` in that we need to remove it